hr_employee_lastnames/models/hr_employee.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.hr_employee_firstname.models.hr_employee import UPDATE_PARTNER_FIELDS
+
+UPDATE_PARTNER_FIELDS += ['lastname2']
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    lastname2 = fields.Char("Second last name")
+
+    @api.model
+    def _get_name_lastnames(self, lastname, firstname, lastname2=None):
+        return self.env['res.partner']._get_computed_name(lastname, firstname, lastname2)
+
+    def _change_values_create(self, vals):
```

Comment:
I like the idea, but I find the name kind of ambiguous. In addition, a docstring would be much appreciated IMHO

Reply:
@luisg123v 

What do you think about new name of methods and docstring?

Reply:
done